CODE_OF_CONDUCT.md

```diff
@@ -74,11 +71,11 @@ Community leaders will follow these Community Impact Guidelines in determining t
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 2.0,
-available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+available at <https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
 
 Community Impact Guidelines were inspired by [Mozilla's code of conduct enforcement ladder](https://github.com/mozilla/diversity).
 
-[homepage]: https://www.contributor-covenant.org
```

Comment:
why moved below?

Reply:
Codacy suggests to have these at the bottom of the file, it should be the standard. 
The place it's used hasn't changed

Reply:
did you check the resulting output? I think we should escape the brackets instead of blindly obey to suggestions :)

Reply:
my fault.